test/ejabberd_tests/tests/mam_SUITE.erl

```diff
@@ -2068,7 +2068,7 @@ check_user_exist(Config) ->
   %% when
   [{_, AdminSpec}] = escalus_users:get_users([admin]),
   [AdminU, AdminS, AdminP] = escalus_users:get_usp(Config, AdminSpec),
-  ok = escalus_ejabberd:rpc(ejabberd_auth, try_register, [AdminU, AdminS, AdminP]),
+  #{} = escalus_ejabberd:rpc(ejabberd_auth, try_register, [AdminU, AdminS, AdminP]),
```

Comment:
I think `ejabberd_auth:try_register` returning an empty map on success is not very intuitive, especially it's not a hook on the outside. Internal hook usage leaks outside the scope of the function now. :(